modules/multiprocess_logging/multiprocess_logging.py

```diff
@@ -0,0 +1,18 @@
+import inspect
+import logging
+
+DEBUG = logging.DEBUG
+INFO = logging.INFO
+WARNING = logging.WARNING
+ERROR = logging.ERROR
+CRITICAL = logging.CRITICAL
```

Comment:
this so so that modules trying to log don't have to import the logging module just for the logging levels

Reply:
Nit: Can we create a separate file and store these values as an enum?